harvest/cli.py

```diff
@@ -265,11 +267,26 @@ def _get_streamer(streamer):
     return streamer_cls()
 
 
-def _get_broker(broker):
+def _get_broker(broker, streamer, streamer_cls):
     """
-    Returns the storage instance specified by the user.
+    Returns the broker instance specified by the user.
     :broker: The type of broker to be instantiated.
     """
+
+    if broker == streamer:
+        debugger.info(f"Using streamer instance as broker. Streamer: {streamer}")
+        return streamer_cls
```

Comment:
if the given broker is also the given streamer, then return the instantiated streamer class.